test/test_proxy_tensor.py

```diff
@@ -581,9 +581,9 @@ def f():
             return bool(val.item() == 2.1)
 
         def test_f():
-            make_fx(f, tracing_mode=self.tracing_mode)()
+            return make_fx(f, tracing_mode=self.tracing_mode)()()
 
-        self.assertRaisesRegex(RuntimeError, "data-dependent", test_f)
```

Comment:
This used to raise error:

`RuntimeError: It appears that you're trying to get value out of a tracing tensor with aten._local_scalar_dense.default - erroring out! It's likely that this is caused by data-dependent control flow or similar.  It may be possible to trace this with dynamic shapes; try setting tracing_mode='symbolic' in your make_fx call.`

but now that the `val` input of `val.item()` actually points to the graph input, we no longer hit this error message.

Is this the right expected behavior?